### PR TITLE
Added account details full screen

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-5127

#### Changes proposed in this Pull Request:

- Added account details full screen
- Added test cases
- Updated docs
- Updated example

#### Screenshots / Screen Recording (if applicable)

- 
Simulator Screenshot - iPhone 12 - 2024-02-28 at 16 34 09

#### To Test:

- Open RegistrationProviderExample.tsx
- Uncomment lines 53-55
- cd login-workflow
- yarn start:example
- Open drawer
- Open Registration Provider

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/example/src/screens/RegistrationSuccessScreen.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export const RegistrationSuccessScreen: React.FC<SuccessScreenProps> = (props) =
     return (
         <SuccessScreenBase
             WorkflowCardHeaderProps={workflowCardHeaderProps}
+            WorkflowCardInstructionProps={{ instructions: 'instructions' }}
```
Comment (ektaghag-eaton) [inline login-workflow/example/src/screens/RegistrationSuccessScreen.tsx:46]: Please remove test code

File: login-workflow/src/screens/AccountDetailsScreen/AccountDetailsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+ * @param lastNameLabel label for the last name text field
+ * @param initialLastName initial value for the last name text field
+ * @param lastNameValidator function that validates the last name text field
+ * @param lastNameTextInputProps props to pass to the last name field
+ * @param WorkflowCardBaseProps props that will be passed to the WorkflowCard component
+ * @param WorkflowCardHeaderProps props that will be passed to the WorkflowCardHeader component
+ * @param WorkflowCardInstructionProps props that will be passed to the WorkflowCardInstructions component
+ * @param WorkflowCardActionsProps props that will be passed to the WorkflowCardActions component
+ * @param errorDisplayConfig configuration for customizing how errors are displayed
+ *
```
Comment (ektaghag-eaton) [inline login-workflow/src/screens/AccountDetailsScreen/AccountDetailsScreen.tsx:26]: Update jsdocs as per BLUI-5309.

**Comment by ektaghag-eaton** [comment]
Please your branch with release/R31.

**Comment by ektaghag-eaton** [comment]
Extra space above the first text input. Recheck UI with https://brightlayer-ui.github.io/patterns/forms#design-specifications

Screenshot 2024-02-29 at 2 12 17 PM

**Comment by ektaghag-eaton** [comment]
Screenshot 2024-02-29 at 2 13 40 PM
Error text is bold. Can you match it with design?

**Comment by ektaghag-eaton** [comment]
RTL, Theme and translations are working fine 👍 .

**Comment by surajeaton** [comment]
@ektaghag-eaton Done. Please review.

File: login-workflow/src/screens/AccountDetailsScreen/AccountDetailsScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+import { useErrorManager } from '../../contexts/ErrorContext';
+import { AccountDetailsScreenBase } from './AccountDetailsScreenBase';
+import { useRegistrationContext } from '../../contexts/RegistrationContext';
+
+/**
+ * Component renders a screen with account details information for support with the application.
+ * Contact information is pulled from the context passed into the workflow.
+ *
+ * @param {AccountDetailsScreenProps} props - Props of Create Account Screen
+ *
```
Comment (ektaghag-eaton) [inline login-workflow/src/screens/AccountDetailsScreen/AccountDetailsScreen.tsx:14]: add @category Component

File: login-workflow/docs/screens/account-details.md [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 A screen that displays text fields to collect the user's basic account details (first name and last name). The AccountDetailsScreen must be used within a `RegistrationContextProvider`.
 
+![Account Details](../../media/screens/account-details.png)
```
Comment (ektaghag-eaton) [inline login-workflow/docs/screens/account-details.md:5]: after fixing UI you need to update image here.

File: login-workflow/src/screens/AccountDetailsScreen/AccountDetailsScreen.tsx (hunk 3 above)
Comment (surajeaton) [reply login-workflow/src/screens/AccountDetailsScreen/AccountDetailsScreen.tsx:14]: It is already there at line 15

**Comment by surajeaton** [comment]
> Screenshot 2024-02-29 at 2 13 40 PM Error text is bold. Can you match it with design?

Updated:
Simulator Screenshot - iPhone 12 - 2024-02-29 at 17 05 41

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**